sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/utils.scala

```diff
@@ -70,7 +70,6 @@ object Utils {
     // Check if we can use TungstenAggregate.
     val usesTungstenAggregate =
       child.sqlContext.conf.unsafeEnabled &&
-      aggregateExpressions.forall(_.aggregateFunction.isInstanceOf[DeclarativeAggregate]) &&
```

Comment:
Can we also remove it from `planAggregateWithOneDistinct`?

For `planAggregateWithoutPartial`, we can add the flag `usesTungstenAggregate` and whenever possible use `TungstenAggregate` (we can do it in a separate pr.).